core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -41,7 +42,7 @@ class CivilizationInfo {
      * Instead, we create a copy list with the change, and replace this list.
      * The other solution, casting toList() every "get", has a performance cost
      */
-    @Transient private var units=listOf<MapUnit>()
```

Comment:
This was private for a reason - so that when getting the civ units you will use getUnits, and you won't edit the list of units directly from outside of the class.
Is there a specific reason you changed this?

Reply:
Not at all, just forget to clean up it, sorry.
It was a leftover after the experiments. Thank you for spotting that.